gradle-sls-packaging/src/main/resources/init.sh

```diff
@@ -99,7 +99,7 @@ stop)
         PID=$(cat $PIDFILE)
         kill $PID
         COUNTER=0
-        while is_process_service $PID $SERVICE && [ "$COUNTER" -lt "240" ]; do
+        while is_process_service $PID $SERVICE && [ "$COUNTER" -lt "180" ]; do
```

Comment:
We've arbitrarily changed the shutdown timeouts in a variety of projects recently, and it's pretty hard to keep track of what the actual source of truth is.

Please could we just pick something a number and put it in the sls-spec, and then make everyone comply with this?  (I think some places might have lower timeouts than 3 minutes)